COLLABORATOR_GUIDE.md

```diff
@@ -542,9 +542,7 @@ Save the file and close the editor. When prompted, enter a new commit message
 for that commit. This is an opportunity to fix commit messages.
 
 * The commit message text must conform to the [commit message guidelines][].
-
-<a name="metadata"></a>
-* Change the original commit message to include metadata. (The
+* <a name="metadata"></a>Change the original commit message to include metadata. (The
```

Comment:
This appeases the linter, but makes reading-the-markdown-as-plain-text harder. I'm not opposed to it, but I wish it wasn't a compromise we had to make. Maybe we can put an "Metadata" header over the list instead and GitHub will auto-generate the #metadata anchor?

Reply:
Yeah, it's not only the linter that has a problem with this, it messes up the list rendering by stopping the list, putting in a paragraph, then creating a new list
```html
<p>Save the file and close the editor. When prompted, enter a new commit message
for that commit. This is an opportunity to fix commit messages.</p>
<ul>
<li>The commit message text must conform to the <a href="/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines">commit message guidelines</a>.</li>
</ul>
<p><a name="user-content-metadata"></a></p>
<ul>
<li>
<p>Change the original commit message to include metadata. (The
<a href="https://github.com/nodejs/node-core-utils/blob/master/docs/git-node.md#git-node-metadata"><code>git node metadata</code></a> command can generate the metadata
for you.)</p>
<ul>
```
I'm not sure how the heading part should work, so maybe you can give me a snippet you'd like

Reply:
It's fine as is. I'd also be OK with removing it.

Looking more closely at the file, doing the header idea I had will introduce other issues that would require other headers and would probably be better in a different PR if we choose to do it.

Reply:
Probably best to leave it as is. Removing it can happen in a subsequent PR if I or someone else feel strongly about it.

Reply:
(By "as is", I mean "as you have it currently in this PR", not "as it is in current master branch".)